src/pybamm/models/full_battery_models/base_battery_model.py

```diff
@@ -337,6 +341,7 @@ def __init__(self, extra_options):
             "voltage as a state": ["false", "true"],
             "working electrode": ["both", "positive"],
             "x-average side reactions": ["false", "true"],
+            "use lumped capacity": ["false", "true"],
```

Comment:
Let's rename this to "use lumped thermal capacity" for clarity, given that batteries also have an charge capacity.